src/World.cpp

```diff
@@ -3933,16 +3938,26 @@ void cWorld::AddQueuedPlayers(void)
 	}  // Lock(m_CSClients)
 
 	// Stream chunks to all eligible clients:
-	for (cPlayerList::iterator itr = PlayersToAdd.begin(), end = PlayersToAdd.end(); itr != end; ++itr)
+	for (auto & AwaitingPlayer : PlayersToAdd)
 	{
-		cClientHandle * Client = (*itr)->GetClientHandle();
+		auto & Player(AwaitingPlayer.first);
```

Comment:
I very much prefer the assignment syntax, rather than construction. At a glance, this is difficult to distinguish from a function call.